Sources/Datadog/Datadog.swift

```diff
@@ -270,10 +271,12 @@ public class Datadog {
                 directories: try obtainTracingFeatureDirectories(),
                 configuration: tracingConfiguration,
                 commonDependencies: commonDependencies,
-                loggingFeatureAdapter: logging.flatMap { LoggingForTracingAdapter(loggingFeature: $0) },
+                loggingFeatureAdapter: logging.map { LoggingForTracingAdapter(loggingFeature: $0) },
```

Comment:
I just wonder, was `map` always possible in such context or has it changed when `compactMap` was introduced?